src/gt4py/backend/dawn_backends.py

```diff
@@ -86,6 +103,20 @@ def _make_global_variables(self, parameters: list, externals: dict):
 
         return global_variables
 
+    def visit_Operator(self, op: enum.Enum):
+        if op in self.OP_TO_CPP:
```

Comment:
A  shorter way to express this: `self.OP_TO_CPP.get(op, op.symbol)`

Reply:
Updated as suggested.